paper.tex

```diff
@@ -617,10 +617,9 @@ \subsubsection{Listings} We aim to use minted to create listings as much as
   \url{https://lean-lang.org/lean4/doc/syntax\_highlight\_in\_latex.html\#example-with-minted}.}
 \end{listing}
 
-The syntax highlighting also works for xDSL-like IRs. Notice that different
-minted styles can be used for different environments. The xDSL environment uses
-the murphy-style in this case, whereas the MLIR version applies the
-colorful-style.
+The syntax highlighting also works for xDSL-like IRs.
+Notice that different minted styles can be used for different environments.
+The xDSL environment uses the murphy-style in this case, whereas the MLIR version applies the colorful-style.
```

Comment:
Originally I had the version info here, so I reflowed the text to have a sentence per line.
However, it feels better down below; regardless, keeping the reflowed text is fine and more readable IMHO.